pkg/share/manila/provisioner.go

```diff
@@ -31,13 +33,17 @@ import (
 
 // Provisioner struct, implements controller.Provisioner interface
 type Provisioner struct {
-	clientset clientset.Interface
+	clientset          clientset.Interface
```

Comment:
Is there a way to avoid interfaces? I already noticed similar behavior in https://github.com/kubernetes/cloud-provider-openstack/pull/788/files#r329300877. From my perspective there is no need to use interfaces, but I may missing something. /cc @adisky